gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/TopologyController.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.rest;
+
+import io.camunda.zeebe.broker.client.api.BrokerClient;
+import io.camunda.zeebe.broker.client.api.BrokerClusterState;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@ZeebeRestController
+public final class TopologyController {
```

Comment:
:thought_balloon: Just a temporary controller, but might be the first we migrate as an example. The idea is all controllers here should be annotated with `@ZeebeRestController`, and not directly with the `@RestController` annotations and so on, to ensure we don't load them if the embedded gateway is disabled.